app/assets/javascripts/discourse/templates/emoji-picker.hbr.erb

```diff
@@ -49,18 +51,22 @@
       </div>
     <% end %>
 
-    {{#if customEmojis.length}}
-      <div class='section' data-section='ungrouped'>
-        <div class='section-header'>
-          <span class="title">{{i18n 'emoji_picker.custom'}}</span>
-        </div>
-        <div class='section-group'>
-          {{#each customEmojis as |emoji|}}
-            <button style="background-url: url("{{emoji.src}}")" type="button" class="emoji" tabindex="-1" title="{{emoji.code}}"></button>
-          {{/each}}
+    {{#each customEmojis as |emojis|}}
+      {{#if emojis.length}}
+        <div class='section' data-section='custom-{{emojis.firstObject.group}}'>
+          <div class='section-header'>
+            <span class="title">
+              {{i18n emojis.firstObject.key}}
+            </span>
+          </div>
+          <div class='section-group'>
+            {{#each emojis as |emoji|}}
+              <button style="background-url: url("{{emoji.src}}")" type="button" class="emoji" tabindex="-1" title="{{emoji.code}}"></button>
```

Comment:
This likely won't pass linting. Is there a way to avoid an inline style?

Reply:
I guess it will because it's not new code :)